examples/src/main/scala/com/basho/riak/spark/examples/dataframes/SimpleScalaRiakDataframesExample.scala

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) 2015 Basho Technologies, Inc.
+ *
+ * This file is provided to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License.  You may obtain
+ * a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.basho.riak.spark.examples.dataframes
+
+import com.basho.riak.client.core.query.indexes.LongIntIndex
+import com.basho.riak.client.core.query.Namespace
+import com.basho.riak.spark._
+import com.basho.riak.spark.util.RiakObjectConversionUtil
+import org.apache.spark.SparkConf
+import org.apache.spark.SparkContext
+import scala.reflect.runtime.universe
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.util.{ Failure, Success }
+import com.basho.riak.client.core.query.RiakObject
+import com.basho.riak.client.api.RiakClient
+import com.basho.riak.client.core.query.Location
+import com.basho.riak.spark.rdd.RiakFunctions
+
+/**
+ * Example shows how Spark DataFrames can be used with Riak
+ * connector for spark
+ */
+object SimpleScalaRiakDataframesExample {
+  private val namespace = new Namespace("users")
+
+  case class UserData(user_id: String, name: String, age: Int, category: String)
+
+  def main(args: Array[String]) {
+    val sparkConf = new SparkConf().setAppName("Riak Spark Dataframes Example")
+
+    setSparkOpt(sparkConf, "spark.master", "local")
+    setSparkOpt(sparkConf, "spark.riak.connection.host", "127.0.0.1:8087")
+
+    val sc = new SparkContext(sparkConf)
+
+    // Work with clear bucket
+    clearBucket(sparkConf)
```

Comment:
To simplify example it's better rewrite as it was done for [SimpleScalaRiakExample.scala](https://github.com/basho/spark-riak-connector/blob/develop/examples/src/main/scala/com/basho/riak/spark/examples/SimpleScalaRiakExample.scala#L60-L66):

```scala

 private val TEST_DATA: String =
    "[" +
        "  {key: 'key-1', indexes: {creationNo: 1}, value: 'value1'}" +
        ", {key: 'key-2', indexes: {creationNo: 2}, value: 'value2'}" +
        ", {key: 'key-3', indexes: {creationNo: 3}, value: 'value3'}" +
        ", {key: 'key-4', indexes: {creationNo: 4}, value: 'value4'}" +
        ", {key: 'key-5', indexes: {creationNo: 5}, value: 'value5'}" +
        ", {key: 'key-6', indexes: {creationNo: 6}, value: 'value6'}" +
    "]"
//...
private def createTestData(sparkConf: SparkConf): Unit = {
    val rf = RiakFunctions(sparkConf)

    rf.withRiakDo(session => {
      rf.createValues(session, SOURCE_DATA, TEST_DATA, true )
    })
  }
```